packages/frontend/src/mantineTheme.ts

```diff
@@ -113,6 +113,11 @@ export const getMantineThemeOverride = (overrides?: {
                                 ),
                             },
                     },
+                    viewport: {
+                        '.only-vertical & > div': {
+                            display: 'block !important', // Only way to override the display value (from `table`) of the Viewport's child element
```

Comment:
I don't understand why would they use `display: table`... any explanations?

Reply:
reason why im asking is that mantine use ScrollArea in their own components like dropdowns and I've seen few bugs because of that...

Reply:
Here is why! https://github.com/radix-ui/primitives/blob/main/packages/react/scroll-area/src/ScrollArea.tsx#L180

```
 {/**
           * `display: table` ensures our content div will match the size of its children in both
           * horizontal and vertical axis so we can determine if scroll width/height changed and
           * recalculate thumb sizes. This doesn't account for children with *percentage*
           * widths that change. We'll wait to see what use-cases consumers come up with there
           * before trying to resolve it.
           */}
```